### PR TITLE
Refs #10388 - Removing license header from models

### PR DESCRIPTION
**Pull request by daviddavis**

### CODE REVIEW

File: app/models/katello/concerns/environment_extensions.rb [hunk 1]
```diff
@@ -1,14 +1,3 @@
-# rubocop:disable AccessModifierIndentation
```
Comment (daviddavis) [inline app/models/katello/concerns/environment_extensions.rb]: :unamused:

**Comment by jlsherrill** [comment]
I want gnu

ACK

**Comment by ehelms** [comment]
The plan here is to do this across every folder as a separate PR? Any reason not to just do it all at once?

**Comment by daviddavis** [comment]
It's more manageable. After a certain number of files, you can no longer see files in the PR diff.

**Comment by daviddavis** [comment]
I'm not doing it per folder per se. Just trying to keep each section below [Github's max](https://help.github.com/articles/what-are-the-limits-for-viewing-content-and-diffs-in-my-repository/). Also, it looks like I've already hit some merge conflicts too (see https://github.com/Katello/katello/pull/5209)

File: app/models/katello/erratum_bugzilla.rb [hunk 2]
```diff
@@ -1,15 +1,4 @@
 #
```
Comment (ehelms) [inline app/models/katello/erratum_bugzilla.rb]: Random empty comment leftover

Comment (ehelms) [inline app/models/katello/erratum_bugzilla.rb]: Looks like a few other files below this are like this as well.

File: app/models/katello/glue/candlepin/activation_key.rb [hunk 3]
```diff
@@ -1,15 +1,3 @@
...
-# This software is licensed to you under the GNU General Public
-# License as published by the Free Software Foundation; either version
-# 2 of the License (GPLv2) or (at your option) any later version.
-# There is NO WARRANTY for this software, express or implied,
-# including the implied warranties of MERCHANTABILITY,
-# NON-INFRINGEMENT, or FITNESS FOR A PARTICULAR PURPOSE. You should
-# have received a copy of GPLv2 along with this software; if not, see
-# http://www.gnu.org/licenses/old-licenses/gpl-2.0.txt.
-
 # rubocop:disable Style/AccessorMethodName
```
Comment (ehelms) [inline app/models/katello/glue/candlepin/activation_key.rb]: In other files, this ends up being at the top with a single blank line before the first piece of code.

File: app/models/katello/key_host_collection.rb [hunk 4]
```diff
@@ -1,15 +1,4 @@
 #
```
Comment (ehelms) [inline app/models/katello/key_host_collection.rb]: Another random comment

File: app/models/katello/repository_docker_image.rb [hunk 5]
```diff
@@ -1,15 +1,4 @@
 #
```
Comment (ehelms) [inline app/models/katello/repository_docker_image.rb]: Got one.

File: app/models/katello/repository_erratum.rb [hunk 6]
```diff
@@ -1,15 +1,4 @@
 #
```
Comment (ehelms) [inline app/models/katello/repository_erratum.rb]: Another

File: app/models/katello/search_favorite.rb [hunk 7]
```diff
@@ -1,15 +1,4 @@
 #
```
Comment (ehelms) [inline app/models/katello/search_favorite.rb]: Another

File: app/models/katello/search_history.rb [hunk 8]
```diff
@@ -1,15 +1,4 @@
 #
```
Comment (ehelms) [inline app/models/katello/search_history.rb]: Another

**Comment by ehelms** [comment]
Hrmm, good point. Take my comments with a grain of salt then, but since I can review them, I did :)

File: app/models/katello/erratum_bugzilla.rb (hunk 2 above)
Comment (daviddavis) [inline app/models/katello/erratum_bugzilla.rb]: Yea, it looks like some of these license headers have an extra commented line at the end.

**Comment by daviddavis** [comment]
@ehelms thanks for the review. I've updated the PR.

File: app/models/katello/glue/candlepin/activation_key.rb (hunk 3 above)
Comment (daviddavis) [inline app/models/katello/glue/candlepin/activation_key.rb]: Looking through the app, it looks like this is the predominant style so I've updated those other files.

**Comment by ehelms** [comment]
APJ

**Pull request merged**